app/controllers/admin/data_exports_controller.rb

```diff
@@ -5,6 +5,22 @@ def new
     @data_export = @export_class.new
   end
 
+  def create
+    # TODO: update to take form params (issue 10017); adding here to create taskable association
+    @data_export = StandardLoanDataExport.create(
+      division_id: division.id,
+      locale_code: "en",
+      start_date: Date.parse("2019-01-01"),
+      end_date: Date.parse("2019-09-01")
+    )
+    Task.create(
+      job_class: DataExportTaskJob,
+      job_type_value: 'data_export_task_job',
+      activity_message_value: 'task_enqueued',
+      taskable: @data_export
```

Comment:
it's feasible to set the taskable association elsewhere if that would be better?